doc/user/filesystem.md

```diff
@@ -169,33 +168,91 @@ Additionally, there are several optional command-line options:
 
 When DFuse starts, it will register a single mount with the kernel, at the
 location specified by the `--mountpoint` option. This mount will be
-visible in `/proc/mounts`, and possibly in the output of `df`.  
-The contents of multiple pools/containers will be accessible via this 
+visible in `/proc/mounts`, and possibly in the output of `df`.
```

Comment:
maybe we could elaborate on what "possibly" means in this context, when will we not see the mount when calling df?